packages/preferences/src/browser/util/preference-tree-generator.spec.ts

```diff
@@ -51,4 +52,50 @@ describe('preference-tree-generator', () => {
         expect(preferenceTreeGenerator['split'](testString)).deep.eq(testString.split(splitter));
     });
 
+    it('PreferenceTreeGenerator.format', () => {
+        const testString = 'aaaBbbCcc Dddd eee';
+        expect(preferenceTreeGenerator['formatString'](testString)).eq('Aaa Bbb Ccc Dddd Eee');
+    });
+
+    describe('PreferenceTreeGenerator.createLeafNode', () => {
+        it('name include semicolon and category parsed correctly when it contain more than one word', () => {
+            const property = 'merge-conflict.autoNavigateNextConflict.enabled';
+            const expectedName = 'Auto Navigate Next Conflict: Enabled';
+            testLeafName(property, expectedName);
+        });
+
+        it('name does not include semicolon and category parsed correctly when it contain more than one word', () => {
```

Comment:
very minor: I think the punctuation mark [':' is a colon](https://en.wikipedia.org/wiki/Colon_(punctuation)) and [';' is a semicolon](https://en.wikipedia.org/wiki/Semicolon), so this (and your references to semicolon below) should be 'name includes / does not include colon'. Otherwise it might be a little confusing.

Reply:
done